simulator-ui/tests/home.spec.ts

```diff
@@ -0,0 +1,169 @@
+import {expect, Page, test} from '@playwright/test';
+
+import {clickOnLinkAndCheckIfTabOpensWithCorrectURL, mockBackendResponse} from "./helpers/helper-functions";
+
+let nbOfSuccessfulTests = 90;
+let nbOfFailedTests = 10;
+let nbOfTotalTests = nbOfSuccessfulTests + nbOfFailedTests;
+
+const scenarioSummariesLinkFilterTriples = [
+  {'testName': 'totalSimulationsButton', 'link': /.*\/scenario-result*/, 'filterText': ''},
+  {'testName': 'successfulSimulationsButton', 'link': /.*\/scenario-result*/, 'filterText': 'SUCCESS'},
+  {'testName': 'failedSimulationsButton', 'link': /.*\/scenario-result*/, 'filterText': 'FAILURE'},
+]
+
+test.beforeEach(async ({page}) => {
+  await mockBackendResponse(page, '**/api/test-results/count-by-status', {
+    "successful": nbOfSuccessfulTests,
+    "failed": nbOfFailedTests,
+    "total": nbOfTotalTests
+  })
+  await page.goto('http://localhost:9000/');
+})
+
+test('should have title, disclaimer, refresh button, reset button, feedback option, summary-tabs and footer', async ({page}) => {
+  const visibleElements: string[] = [
+    'disclaimer',
+    'refreshListButton',
+    'resetButton',
+    'feedbackStarGithub',
+    'footer',
+    'totalSimulationsPercentage',
+    'successfulSimulationsPercentage',
+    'failedSimulationsPercentage'
+  ]
+
+  await expect(page).toHaveTitle(/Citrus Simulator/);
+  for (const element of visibleElements) {
+    await expect(page.getByTestId(element)).toBeVisible();
+  }
+});
+
+test('check if summary-tab displays right percentage with round numbers', async ({page}) => {
+  await checkIfSummaryTabsAreDisplayingRightNumbersFunction(page, nbOfTotalTests, nbOfSuccessfulTests, nbOfFailedTests);
+})
+
+test('total, successful, failed tabs should display percentage in simulations count rounded to two decimal numbers', async ({page}) => {
+  const successfulTestsBig = 746039;
+  const failedTestsBig = 490;
+  const totalTestsBig = successfulTestsBig + failedTestsBig;
+
+  await mockBackendResponse(page, '**/api/test-results/count-by-status', {
+    "successful": successfulTestsBig,
+    "failed": failedTestsBig,
+    "total": totalTestsBig
+  })
+  await page.goto('http://localhost:9000/');
+
+  await checkIfSummaryTabsAreDisplayingRightNumbersFunction(page, totalTestsBig, successfulTestsBig, failedTestsBig);
+});
+
+//should there be two separate tests for this? Or a for loop?
+test('should move to right page with feedback resp suggestion link', async ({page}) => {
+  await clickOnLinkAndCheckIfTabOpensWithCorrectURL(page, 'feedbackLinkStarGithub', /.*\/github.com\/citrusframework\/citrus-simulator/);
+  await clickOnLinkAndCheckIfTabOpensWithCorrectURL(page, 'feedbackAndSuggestionLink', /.*\/github.com*/); //could be the login site ore the new-issue site: should one test the two options?
```

Comment:
one could land on the login site ore the new-issue site: should one test the two options?